wp-content/themes/reconasia/assets/_scss/pages/home.scss

```diff
@@ -2,162 +2,370 @@
+        position: absolute;
+        top: 0;
+        left: 0;
+        height: 100%;
+        object-fit: cover;
+        @include z-index(hpHeaderImg);
+      }
+    }
+  }
+
+  &__featured-secondary {
+    grid-area: featured;
+    background: $color-bg-light-100;
+    @include full-width-background($color-bg-light-100);
+
     @include breakpoint('small') {
-      --padding-top: #{rem(24)};
+      padding-right: rem(24);
+      padding-left: rem(24);
+      @include full-width-background(transparent);
     }
 
     @include breakpoint('large') {
-      --padding-sides: var(--padding-top);
-      grid-column: 2;
+      margin-right: rem(48);
     }
 
-    @include breakpoint('xlarge') {
-      --padding-top: #{rem(32)};
+    .post-block {
+      padding-bottom: rem(24);
+
+      &:not(:last-child) {
+        border-bottom: 1px solid $color-black-130;
+      }
+
+      @include breakpoint('xlarge') {
+        padding-bottom: rem(32);
+      }
+
+      &:first-child {
+        display: flex;
+        flex-direction: column;
+        margin-top: 0;
+        padding-top: rem(40);
+
+        @include breakpoint('small') {
+          margin-top: rem(40);
+        }
+
+        /* stylelint-disable-next-line */
+        .post-block__img {
+          @include breakpoint('xlarge') {
+            margin-bottom: rem(40);
+          }
+
+          /* stylelint-disable-next-line */
+          img {
+            width: 100%;
+          }
+        }
+      }
+
+      &:not(:last-child) {
+        margin-bottom: rem(40);
+      }
     }
+  }
 
-    &-title {
-      grid-area: title;
-      margin-bottom: rem(12);
-      // @extend %font-heading-3-xl-serif;
+  &__cta {
+    grid-area: cta;
+    padding-top: rem(40);
+    padding-bottom: rem(64);
+    background: $color-bg-light-200;
+    @include full-width-background($color-bg-light-200, '', false);
+
+    .icon {
+      margin-left: rem(8);
     }
+  }
 
-    &-desc {
-      grid-area: desc;
-      margin-bottom: rem(24);
-      // @extend %font-body-md;
+  &__featured-content {
+    @include csis-block-full-width-wrapper();
+    grid-area: content;
+    padding-top: rem(20);
+    padding-bottom: rem(20);
+
+    @include breakpoint('large') {
+      padding-top: rem(48);
+      padding-bottom: rem(60);
     }
 
-    &-img {
-      display: block;
-      grid-area: img;
-      margin-bottom: rem(24);
+    @include breakpoint('xxlarge') {
+      padding-left: rem(80);
     }
 
-    &-articles {
-      grid-area: articles;
-      margin-bottom: var(--padding-top);
+    &-wrapper {
+      margin-right: 6vw;
+      margin-left: 6vw;
+      padding: rem(40);
+      color: $color-white-100;
+      background-color: $color-bg-dark-290;
 
-      > *:not(:first-child) .post-block__excerpt {
-        display: none;
+      @include breakpoint('large') {
+        width: 44vw;
+        max-width: rem(800);
+      }
+
+      @include breakpoint('xxlarge') {
+        margin-left: calc((100vw - 1400px) / 2);
       }
```

Comment:
You were almost there! These are still misaligned by ~15px because your margin left does not take into account the scrollbar. Update to:

margin-left: calc(((100vw - 1400px)  / 2) - var(--scrollbarX));